src/elements/Checkbox/Checkbox.tsx

```diff
@@ -1,120 +1,104 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import { KeyboardEvent, MouseEvent, ReactElement, ReactNode } from "react";
-import { InputLabel } from "@elements/InputLabel/InputLabel";
 import IconCheck from "@elements/Icon/Generated/IconCheck";
 import IconMinus from "@elements/Icon/Generated/IconMinus";
-import { Style } from "@utilities/enum";
-import generateRandomId from "@utilities/generateRandomId";
+import { useCheckbox } from "@react-aria/checkbox";
+import { useFocusRing } from "@react-aria/focus";
+import { useToggleState } from "@react-stately/toggle";
+import { merge } from "@utilities/merge";
+import { FC, useRef } from "react";
 
-export enum CheckboxSelectionState {
-    Unselected = "Unselected",
-    Selected = "Selected",
-    Indeterminate = "Indeterminate",
+export enum Style {
```

Comment:
```suggestion
export enum CheckboxStyle {
```
Otherwise we have collision with `enum.ts`